source/llassetgen-rendering/main.cpp

```diff
@@ -69,10 +69,10 @@ class Window : public WindowQt {
 
     virtual ~Window() {}
 
-    virtual void initializeGL() override {
+    virtual bool initializeGL() override {
         globjects::init([this](const char * name) {
```

Comment:
With current glbinding master, this line will get a problem.
Have a look at https://github.com/cginternals/glbinding/compare/8c18e53faffc4bfda11901c573a5c4bb0fc0b2d3...beac98e46babb819d225dee7bfd9b13774569351 for a workaround.

Reply:
What exactly is the problem? This is `globjects::init`, which does not appear in your workaround. Please help me to understand.

Reply:
This language construct dismisses the support for MSVC 2013. As we want to support this compiler further, we were forced to degrade to plain C function pointers for the lookup callback. Unfortunately, the present implementation requires a binding of this into the context of the callback that is not possible with plain C function pointers. A workaround is the usage of a global/static variable (as outlined in my reference to glbinding).

Reply:
thank you for this explanation, very helpful :)